content/blog/metafacture-releases_core-6.2.0_fix-1.2.0/index.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Metafacture releases: Core 6.2.0, Fix 1.2.0"
+date: "2025-01-24T16:15:03.2300Z"
+description: "Changes coming with the releases of metafacture-core 6.2.0 and
+metafacture-fix 1.2.0"
+authors: [{lastname: "Christoph",
+           firstname: "Pascal"}]
+---
+
+*Preamble*
+
+This will be the last time `metafacture-fix` will be released on its own as it will be [incorporated into `metafacture-core`](https://github.com/metafacture/metafacture-core/issues/577).
+
+This post describes the new developments coming with:
+
+a) [`metafacture-core release
+    6.2.0`](https://central.sonatype.com/namespace/org.metafacture) since the [release of metafacture-core 6.1.2](https://blog.metafacture.org/metafacture-releases_core-6.1.2_fix-1.1.2_playground-1.1.2/#metafacture-core) in July 2024.
```

Comment:
```suggestion
a) [`metafacture-core release 6.2.0`](https://central.sonatype.com/namespace/org.metafacture) since the [release of metafacture-core 6.1.2](https://blog.metafacture.org/metafacture-releases_core-6.1.2_fix-1.1.2_playground-1.1.2/#metafacture-core) in July 2024.
```